util/regexp-assemble/data/942362.data

```diff
@@ -3,138 +3,161 @@
 
 ##!+ i
 
-alter\s+char\s?\(?
-alter\s+group_concat\s?\(?
-alter\s+load_file\s?\(?
-create\s+char\s?\(?
-create\s+group_concat\s?\(?
-create\s+load_file\s?\(?
-delete\s+char\s?\(?
-delete\s+group_concat\s?\(?
-delete\s+load_file\s?\(?
-desc\s+char\s?\(?
-desc\s+group_concat\s?\(?
-desc\s+load_file\s?\(?
-insert\s+char\s?\(?
-insert\s+group_concat\s?\(?
-insert\s+load_file\s?\(?
-load\s+char\s?\(?
-load\s+group_concat\s?\(?
-load\s+load_file\s?\(?
-rename\s+char\s?\(?
-rename\s+group_concat\s?\(?
-rename\s+load_file\s?\(?
-select\s+char\s?\(?
-select\s+group_concat\s?\(?
-select\s+load_file\s?\(?
-truncate\s+char\s?\(?
-truncate\s+group_concat\s?\(?
-truncate\s+load_file\s?\(?
-update\s+char\s?\(?
-update\s+group_concat\s?\(?
-update\s+load_file\s?\(?
+##! Helpers
+
+##!> template cmd_alter    alter
+##!> template cmd_create   create
+##!> template cmd_delete   delete
+##!> template cmd_desc     desc
+##!> template cmd_insert   insert
+##!> template cmd_load     load
+##!> template cmd_rename   rename
+##!> template cmd_select   select
+##!> template cmd_truncate truncate
+##!> template cmd_update   update
+
+##!> template seq_char         \s+char\s?\(?
+##!> template seq_group_concat \s+group_concat\s?\(?
+##!> template seq_load_file    \s+load_file\s?\(?
+
+##!> template cmd_drop     drop
+
+##!> template nonword_number ^[\W\d]+\s*?
+##!> template nonword_number_cmd_alter ^[\W\d]+\s*?alter\s*
+##!> template cmd_alter_space alter\s*
```

Comment:
Remove this template and just use `cmd_alter`. You do the same for `{{cmd_create}}\s+\w+` etc.

Reply:
Done.